arrow-refined-types/README.md

```diff
@@ -0,0 +1,199 @@
+# Module arrow-refined-types
+
+# Package arrow.refinement
+
+A refined type is any regular type constrained by predicates expected to hold in all possible values of the type's
+constructor.
+
+The Arrow refined-types plugin verifies at compile and runtime refined types constructor calls, ensuring values in
+declared types truly represent the programmers intent.
+
+## Setup.
+
+The refined-types plugin is a Kotlin compiler plugin that can be enabled via Gradle in your build by adding it to the
+list of your Gradle plugins.
+
+Release
+
+```groovy
+plugins {
+    id "io.arrow-kt.arrow-refined-types" version "<release-version>"
+}
+```
+
+Snapshots
+
+```groovy
+buildscript {
+    repositories {
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+    }
+    dependencies {
+        classpath "io.arrow-kt:gradle-plugin:<snapshot-version>"
```

Comment:
```suggestion
        classpath "io.arrow-kt:arrow-refined-types-gradle-plugin:<snapshot-version>"
```

Reply:
Temporarily:

```
buildscript {
    repositories {
        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
    }
    dependencies {
        classpath "io.arrow-kt:gradle-plugin:<snapshot-version>"
    }
}

apply plugin: "io.arrow-kt.arrow"

arrowMeta.plugins = ['refined-types']
```

It will be fixed very soon!! :raised_hands: